expanded.html

```diff
@@ -30,7 +30,7 @@ <h1>
       <nav id="nav">
         <ul>
           <li>
-            <a href="#" onClick="MyWindow=window.open('fields-expanded.html','MyWindow','width=600,height=600'); return false;">Field Explanations</a>
```

Comment:
I like it opening in a new tab so you don't lose your filters. Probs a better way to do that though eh